example/real-world/Render/GroupForm.purs

```diff
@@ -50,22 +51,34 @@ renderName =
   }
 
 renderSecretKey1 :: FormlessState -> FormlessHTML
-renderSecretKey1 =
-  UI.formlessField UI.input
-  { label: "Secret Key 1"
-  , help: "Provide a secret identifier for the group."
-  , placeholder: "iasncat3ihba/0"
-  , sym: prx.secretKey1
-  }
+renderSecretKey1 st =
+ UI.input
+   { label: "Secret Key 1"
+   , help: UI.resultToHelp
+             "Provide a secret identifier for the group"
+             (F.getResult prx.secretKey1 st.form)
+   , placeholder: "ia30<>Psncdi3b#$<0423"
+   }
+   [ HP.value $ F.getInput prx.secretKey1 st.form
+   , HE.onValueInput $ HE.input \str -> F.AndThen
+       (F.modifyValidate_ prx.secretKey1 str)
+       (F.validate_ prx.secretKey2)
```

Comment:
This is an example of how you can make sure dependent fields are also validated.